zer0-os/zOS
# Enable browser level notifications

### What does this do?

Publishes Pusher based Messenger Conversation type notifications to the browser.

### How do I test this?


- Verify that Browser Notifications are enabled for the site
- Connect to zOS as two different users in two different browser sessions
- Start a Messenger Conversation between the two users
- Mention either one of the users in the conversation
- Verify that a Browser notification appears

### Screen capture


https://user-images.githubusercontent.com/31045/233151462-e120bc65-7ed4-4719-966b-5d3320a2d131.mp4
